Dynatrace/helm-charts
# Add merged index file to github

# Description

Currently we have 2 repos with different helm indices. This PR merges the index of the new repo with the index of the old one.

## How can this be tested?

Add the old helm-repo to your helm installation and install the newest dynatrace operator (>=0.5.0). This should work now.

## Checklist
